mixin-styles-gb/block-templates/404.html

```diff
@@ -0,0 +1,35 @@
+<!-- wp:group {"className":"mxs-wrapper"} -->
+<div class="wp-block-group mxs-wrapper"><div class="wp-block-group__inner-container"><!-- wp:template-part {"slug":"header","theme":"mixin-styles-gb","tagName":"header","align":"full","className":"mxs-header mxs-header\u002d\u002dmain"} /-->
```

Comment:
Remove the inner-container from all group blocks. This is related to https://github.com/WordPress/gutenberg/pull/29335